repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/ipuser/IpUserTest.groovy

```diff
@@ -89,4 +93,23 @@ class IpUserTest extends ReposeValveTest {
         !group.contains("local-lan-ip;q=0.6")
     }
 
+    def "should fail to start with bad config"() {
```

Comment:
I wonder if this should be broken out into its own test, for the sake of consistency.